avalanche/spec.md

```diff
@@ -183,3 +197,11 @@ When `Rejected` transactions are finalized they should be removed from the mempo
 
 Because the event loop is firing off requests every 10ms it will send more requests than are required to finalized the transaction. If any responses arrive
 after the transaction has been finalized they should be ignored and not affect the state of the transaction.
+
+> This architecture has censorship issues
+> In BTC, nodes can attempt to censor transactions, by orphaning a block but they risk losing money if the majority does not also censor the tx.
+> And a clear record of the censorship exists in the form of the orphaned chain, so although the money did not move, in an important sense censorship cannot happen, since censorship is most effective when invisible.
+> But it seems like with this voting protocol a majority of nodes can vote "Rejected" to quite silently implement censorship.
```

Comment:
Yes, so this is another TBD part. We've discussed making a `No` vote just mean `not-finalized` and not `invalid`. So txs voted `No` could still be included in the chain, but only double spends of txs voted `Yes` would be considered invalid. Of course, this could have other ramifications and possibly create edge cases. This is something that needs to be thought through really well.

Reply:
I agree that the only thing that matters is a multi-spend having a "yes" vote (although all txs need to be considered). "No" only matters if that tx is a multi-spend of a tx finalized as "yes".